packages/tic-tac-toe/ember-cli-build.js

```diff
@@ -4,6 +4,9 @@ const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 const {getNetworkName, setupGanache} = require('@statechannels/devtools');
 const {deploy} = require('./deployment/deploy');
 
+const environment = EmberApp.env();
+const IS_DEV = environment === 'development';
```

Comment:
Nit: should this be `isDev`?

Reply:
I have always named these consts like this in every Ember project I have worked on 🤷‍♂